kong/init.lua

```diff
@@ -651,12 +629,25 @@ function Kong.init_worker()
 
   kong.db:set_events_handler(worker_events)
 
-  ok, err = load_declarative_config(kong.configuration,
-                                    declarative_entities,
-                                    declarative_meta)
-  if not ok then
-    stash_init_worker_error("failed to load declarative config file: " .. err)
-    return
+  if kong.configuration.database == "off" then
+    local t = lmdb_txn.begin(1)
```

Comment:
let's have a comment for this (or even better make this a function like sanity_test())